cloud-formation/dev-template.json

```diff
@@ -58,7 +64,25 @@
             }
         },
         "ImageBucket": {
-            "Type": "AWS::S3::Bucket"
+            "Type": "AWS::S3::Bucket",
+            "Properties": {
+                "CorsConfiguration": {
+                    "CorsRules": [
+                        {
+                            "AllowedMethods": [
+                                "GET"
+                            ],
+                            "AllowedOrigins": [
+                                "*"
+                            ],
+                            "ExposedHeaders": [
+                                "Date"
+                            ],
+                            "MaxAge": "3600"
```

Comment:
It doesn't - but this was a copy paste muck up.
If we leave them out it respects the normal settings.